### DISCUSSION TITLE
Sort out permissions for GitHuB Discussions

### DISCUSSION THREAD

**Discussion by kelliedesigner** [ministryofjustice/moj-frontend#252] [EXPERIMENTAL: Components, pages and patterns]
Currently designers aren't able to edit the main post of Discussions, so they feel they can't contribute.

**Comment by gregtyler** [comment]
Does this apply to members of the ["designers" GitHub team](https://github.com/orgs/ministryofjustice/teams/designers/members)? If _they_ can't edit main posts then I don't think there's anything we can do because it would require giving full admin rights to everyone.
